### DISCUSSION TITLE
The badges feel pretty generic.

### DISCUSSION THREAD

**Discussion by meatnordrink** [community/community#46135] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Just some feedback, since the badges are in beta - most of them feel pretty generic. I'm not opposed to badges in general, but it seems like every developer has the same set. Who hasn't got code in the Artic Vault? And pretty much any developer using GitHub has code that's been merged ("shark"). 

It would just be nice if they actually denoted something meaningful about the developer's behavior. Currently it feels like every developer has basically the same set.}

**Comment by jge162** [comment]
Badges are cool but stuff for people who joined 5 years ago 10 years ago should get maybe a unique badging for tenure. That would be nice.
